scripts/build_python.sh

```diff
@@ -60,16 +71,16 @@ Input Options:
                                                             By default it is minimal.
   -t --time_between_case_retries MRPActiveRetryInterval     Specify MRPActiveRetryInterval value
                                                             Default is 300 ms
-  -i, --install_virtual_env <path>                          Create a virtual environment with the wheels installed
-                                                            <path> represents where the virtual environment is to be created.
-  -c, --clean_virtual_env  <yes|no>                         When installing a virtual environment, create/clean it first.
-                                                            Defaults to yes.
-  --include_pytest_deps  <yes|no>                           Install requirements.txt for running scripts/tests and
-                                                            src/python_testing scripts.
-                                                            Defaults to yes.
-  -j, --jupyter-lab                                         Install jupyterlab requirements.
-  --extra_packages PACKAGES                                 Install extra Python packages from PyPI
-  -z --pregen_dir DIRECTORY                                 Directory where generated zap files have been pre-generated.
+  -e, --create-virtual-env  <path>                          Create Python virtual environment in the specified path
+  -c, --clean-virtual-env   <yes|no>                        Clean the virtual environment if it exists; defaults to yes
+  -i, --install-wheels                                      Install generated wheels; this option is enabled by default
```

Comment:
This will create a conflict with people copy & pasting in history. It is inconvenient for people to remember what version they checked out for commands to work. How about keeping `-i/--install-virtual-env` and maybe `-w/--install-wheels` for the new option?